src/helpers/mapping.js

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2018  Shivam Tripathi
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+
+export const musicBrainzReleaseId = 1;
```

Comment:
Yeah, I have written couple of mapping functions on the bb-data side and plan to use them to get good mapping which is done only once before the consumers/producers start off. This was primarily done by me to get things working straight away, so that flow of work isn't shifted.
I'll be making the mapping functions on bb-data side as well as bb-import tonight most probably.